lxd/node/config.go

```diff
@@ -169,16 +168,53 @@ var ConfigSchema = config.Schema{
 	"storage.images_volume":  {},
 }
 
+func validateCoreHTTPSAddress(value string) error {
+	if value == "" {
+		return nil // Deleting entry
+	}
+
+	host := value
+	splitHost, _, err := net.SplitHostPort(value)
+	if err == nil {
+		host = splitHost
+	}
+
+	ip := net.ParseIP(host)
+	if ip != nil {
+		return nil
+	}
+
+	_, err = net.LookupHost(host)
+	if err == nil {
+		return nil
+	}
+
+	return fmt.Errorf("Only IPV4, IPV6 address, :<HOST>:<PORT>, or a valid and resolvable Hostname are valid")
+}
+
 func validateClusterHTTPSAddress(value string) error {
 	if value == "" {
 		return nil // Deleting entry
 	}
+
 	host, _, err := net.SplitHostPort(value)
 	if err != nil {
 		return errors.Wrap(err, "Address not in form of <HOST>:<PORT>")
 	}
+
 	if shared.StringInSlice(host, []string{"[::]", "0.0.0.0"}) {
 		return fmt.Errorf("Invalid IP address or DNS name")
 	}
-	return nil
+
+	ip := net.ParseIP(host)
+	if ip != nil {
+		return nil
+	}
+
+	_, err = net.LookupHost(host)
+	if err == nil {
+		return nil
+	}
+
+	return fmt.Errorf("Only valid IPV4, IPV6 aaddresses or resolvable hostnames are valid")
```

Comment:
Why is this error different from the one in validateCoreHTTPSAddress, even though in this function higher up it has an error that says `errors.Wrap(err, "Address not in form of <HOST>:<PORT>")`? Seems inconsistent in messaging. Do we only accept an IP (and no port here)? If so, then the error above needs changing.